File: ARCADE/Intro/Level 8_2: firstDigit.cs
    using System;
     
    public class Test
    {
    	public static char firstDigit(string inputString){
            int count = 0;
            for (int i = 0; i < inputString.Length; i++){
                if ('0' > inputString[i] || inputString[i] > '9') count++;
                else break;
            }
            return inputString[count];
        }
    	public static void Main()
    	{
    		string inputString = "var_1__Int";
    		Console.WriteLine(firstDigit(inputString));
     
    		string inputString1 = "q2q-q";
    		Console.WriteLine(firstDigit(inputString1));
    	}
    }

File: ARCADE/Intro/Level 8_2: firstDigit.pas
program firstDigit;
 
function firstDigit(inputString: string): char;
var i : integer;
    rs : char;
begin
    rs := '_';
    for i := 1 to length(inputString) do
        if((48 <= ord(inputString[i])) and (ord(inputString[i]) <= 57)) then
            begin
                rs := inputString[i];
                break;
            end;
 
    firstDigit := rs;        
end;
 
var inputString : string;
begin
	inputString := 'var_1__Int';
	writeln(firstDigit(inputString));
	inputString := 'q2q-q';
	writeln(firstDigit(inputString));
	inputString := '0ss';
	writeln(firstDigit(inputString));
end.
